dev/integration_tests/flavors/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -531,6 +713,17 @@
 			defaultConfigurationIsVisible = 0;
 			defaultConfigurationName = "Release Free";
 		};
+		F723D23C2464E1C10004F0E0 /* Build configuration list for PBXNativeTarget "Paid App" */ = {
+			isa = XCConfigurationList;
+			buildConfigurations = (
+				F723D23D2464E1C10004F0E0 /* Debug Free */,
+				F723D23E2464E1C10004F0E0 /* Debug Paid */,
+				F723D23F2464E1C10004F0E0 /* Release Free */,
+				F723D2402464E1C10004F0E0 /* Release Paid */,
```

Comment:
Requiring these permutations of build configurations is still a bit icky, which is what the rest of https://github.com/flutter/flutter/issues/14648 can track.